java/src/jmri/jmrix/rps/serial/SerialAdapter.java

```diff
@@ -314,7 +291,7 @@ void handleIncomingData() throws java.io.IOException {
                     break;
                 }
                 // Strip off the CR and LF
-                if (char1 != 10 && char1 != 13) {
```

Comment:
@silverailscolo Thanks for all the cleanup!  

From the comment, this is intended to remove either CR or LF, so the && should probably have been a ||

Reply:
My IDE told me one of these would never be true, ``not 10 && not 13`` seems fine if it would have been.

Reply:
Aha!  Earlier, the code had already handled 13.  Thanks!